core/src/main/resources/error/README.md

```diff
@@ -66,6 +66,8 @@ To access error fields, catch exceptions that extend `org.apache.spark.SparkThro
 
 Error classes are a succinct, human-readable representation of the error category.
 
+An uncategorized errors can be assigned to a legacy error class with the prefix _LEGACY_ERROR_TEMP_ and an unused sequential number, for instance _LEGACY_ERROR_TEMP_053.
```

Comment:
nit: Maybe we should using `\` in front of `_LEGACY_ERROR_TEMP_` as an escape character ??

Otherwise, maybe it should look something like: _LEGACY_ERROR_TEMP_ in the documents.

Reply:
I will mark it as code. Thank you.

Reply:
Sounds good!